sygnal/sygnal.py

```diff
@@ -271,14 +299,16 @@ def check_section(section_name, known_keys, cfgpart=config):
     check_section("sentry", {"enabled", "dsn"}, cfgpart=config["metrics"])
 
 
-def merge_left_with_defaults(defaults, loaded_config):
+def merge_left_with_defaults(
```

Comment:
I haven't looked very carefully, but 

```
merged = merge_left_with_defaults(defaults, loaded_config)
```

sounds to me like it would be the same as the following two lines:

```
merged = copy.deepcopy(loaded_config)
merged.update(defaults)
```

Let's not change this. Just mentioning it for interest!